wagtail/admin/templates/wagtailadmin/shared/buttons.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Pattern, generateDocs } from 'storybook-django/src/react';
+
+export default {
+    title: 'Shared / buttons',
+  };
+
+export const base = (args) => (
+    <section id="buttons">
+    <h2>Buttons</h2>
+    <h3>Basic buttons</h3>
+
+    <a href="#" className="button">button link</a>
+    <button className="button">button element</button>
+
+    <h3>Secondary buttons</h3>
+
+    <a href="#" className="button button-secondary">button link</a>
+    <button className="button button-secondary">button element</button>
+
+    <h3>Small buttons</h3>
+
+    <a href="#" className="button button-small">button link</a>
+    <button className="button button-small">button element</button>
+
+    <h4>(secondary)</h4>
+    <a href="#" className="button button-small button-secondary">button link</a>
+    <button className="button button-small button-secondary">button element</button>
+
+    <h3>Disabled buttons</h3>
+
+    <a href="#" className="button disabled">button link</a>
+    <button className="button button-small disabled">button element</button>
+
+    <h3>Bi-color icon buttons with text</h3>
+{/* 
```

Comment:
I would just remove this for now.
Or investigate how the icons work for the wagtail/admin/templates/wagtailadmin/shared/icons.stories.tsx